kdl/normalize.kdl

```diff
@@ -0,0 +1,14 @@
+ctr {Pair a b}
```

Comment:
It would be good to have a test in `kindelia/cli/tests` that test the expected output for the cli command `kindelia test normalize.kdl`. I thought that we already had a test like this, but I saw that we didn't. Maybe i will do this later for this and other files from `kdl` folder